src/java/org/apache/cassandra/index/sai/plan/Expression.java

```diff
@@ -345,6 +345,7 @@ public boolean isSatisfiedBy(ByteBuffer columnValue)
     /**
      * Returns the lower bound of the expression as a ByteComparable with an encoding based on the version and the
      * validator.
+     *
```

Comment:
Those were not intentionally added... something from the IDE formatting, I will revert